src/libraries/Common/tests/System/Net/Http/HttpClientHandlerTest.ServerCertificates.cs

```diff
@@ -228,14 +228,38 @@ public async Task NoCallback_BadCertificate_ThrowsException(string url)
         }
 
         [OuterLoop("Uses external servers")]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/77726")]
         [ConditionalFact(nameof(ClientSupportsDHECipherSuites))]
         public async Task NoCallback_RevokedCertificate_NoRevocationChecking_Succeeds()
```

Comment:
Name of the test now seems not fully correct, as we do have a callback now 😄

Reply:
I have a feeling this group of tests starting with "NoCallback_" was supposed to validate HttpClient's default behavior... so can we have both tests? The one without callback for SocketsHttpHandler (which should technically cover HttpClientHandler without callback on platforms where it has SocketsHttpHandler inside), and the one with the callback you've added for general HttpClientHandler?

Reply:
One way how to solve this is removing dependency on `badssl.com`. It may need  some more infrastructure changes but perhaps it is better fix.